### ISSUE TITLE
Look into integrating gamebox into this project.

### ISSUE THREAD

**Issue by gilles-leblanc** [gilles-leblanc/gameproject#28]
https://github.com/shawn42/gamebox is a game template for Gosu. Check if it can be applied to this project to save time. If so, implement it.

**Comment by gilles-leblanc** [comment]
I have started looking into this.

**Comment by shawn42** [comment]
Great! I noticed the branch. The docs are in need of updating, I think the best way to wrap your head around the basics of Gamebox is by looking at a couple of examples.
https://github.com/shawn42/killbox
https://github.com/shawn42/omg_aliens

I tend to hang out in the #gamebox IRC channel on freenode and will respond pretty quickly to email questions.

**Comment by gilles-leblanc** [comment]
I closed this with the wrong commit. 

Anyway while Gamebox seems great there was a lot to do to change the current game and I wasn't seeing the gain for doing this which affected my motivation to make the change. So I intended to close this issue anyway.

It looks very useful for developing games but I would have to know what it would change concretely for me before investing the effort to change this already started project.
